modules/querying/pages/query-optimizer/enable-cost-optimizer.adoc

```diff
@@ -0,0 +1,108 @@
+= Enabling Query Optimizer
```

Comment:
I don't think we are being overt about this feature. It is a new feature, and we are introducing it like any other new feature. "Aiding Query Optimizer" complicates the subject for users because before 3.7, the term Query Optimizer was never used, which makes sense, because we are using it strictly in the sense of cost-based optimization.